zone/matches.php

```diff
@@ -899,4 +899,14 @@ public function draw(int $user_id): array
             return $match_ids;
         });
     }
+
+    //TODO: eru....can we do it like that? This looks way easier than the complicated sql stuff you are doing....
```

Comment:
Looks fine to me, but I don't know what complicated sql stuff you're referring to.